docs/addons.md

```diff
@@ -74,7 +74,7 @@ either remove this installation prior to enabling this addon, or mark cert-mange
 As long as you are using v1 versions of the cert-manager resources, it is safe to remove existing installs and replace it with this addon**
 
 ##### Self-provisioned cert-manager
-{{ kops_feature_table(kops_added_default='1.21', k8s_min='1.16') }}
+{{ kops_feature_table(kops_added_default='1.20', k8s_min='1.16') }}
```

Comment:
Shouldn't this be 1.20.2?

Reply:
Technically yes, but I did not see any patch versions being mentioned so far. Will gladly add it, if this is okay.

Reply:
```suggestion
{{ kops_feature_table(kops_added_default='1.20.2', k8s_min='1.16') }}
```